### ISSUE TITLE
After working for a few days all devices went offline in 1.0.4

### ISSUE THREAD

**Issue by glowfishDE** [homebridge/homebridge#2564]
Hi,

using the actual 1.0.4 version for a few days now which worked great, all my devices went offline yesterday.

Funny thing is, that some get a status and most dont. After a while some others get a status while others wont work.

My doorbell says "not reachable" but when i push the ring button at my door, the iphone sends me a push alert that the door bell was triggered.

So its totally strange. I can control some things and others i cant, but most devices say "not reachable".

I tried restarting everything (appleTV, whole synology NAS with the OZNU Docker Container, the homematic raspberry PI as well as the whole internet router) but it did not help.

I´m kinda clueless about it. All logs show no errors.

Update: When i click on a device it updates. Seeing that some update and some dont, it looks like a cache or memory issue to me. Like the app is not able to refresh/download so many status infos in total. I have ~ 85 devices

After updating a few devices manually by clicking them, all "not reachable" devices tried to update too automatically, but all fail with "no response" again.

**Comment by darrenmcuk** [comment]
Same problem here...just tried updating to the latest NodeJS seems to have some difference but just waiting to fail again.

**Comment by dsadsa897897r** [comment]
I have the same issue

**Comment by darrenmcuk** [comment]
Can’t seem to understand what has changed but something has

**Comment by dsadsa897897r** [comment]
When I removed my plugin "accessory": "Script2"  I managed to get it all working again

**Comment by ramirocaballero** [comment]
Same problem here!

**Comment by MikeDeltaHH** [comment]
Same problem here since a few hours ago! The homebrigde shows no error in the logs and i can control all devices from the homebrdige itself but in homekit all devices are offline! I´ve made no changes in the last days and have absolute no idea what´s happen!

With debug log on i have the following entries:

EventedHTTPServer Server listening on port 51826 +0ms
[2020-5-11 21:09:49] Homebridge is running on port 51826.
EventedHTTPServer [::ffff:192.168.2.43] New connection from client +1s
EventedHTTPServer [::ffff:192.168.2.43] HTTP server listening on port 44401 +1ms
EventedHTTPServer [::ffff:192.168.2.43] HTTP request: /pair-verify +21ms
HAPServer [CC:22:3D:E3:CE:30] HAP Request: POST /pair-verify +0ms
HAPServer [CC:22:3D:E3:CE:30] Pair verify step 1/2 +2ms
EventedHTTPServer [::ffff:192.168.2.43] HTTP Response is finished +29ms
EventedHTTPServer [::ffff:192.168.2.43] HTTP request: /pair-verify +12ms
HAPServer [CC:22:3D:E3:CE:30] HAP Request: POST /pair-verify +38ms
HAPServer [CC:22:3D:E3:CE:30] Pair verify step 2/2 +1ms
HAPServer [CC:22:3D:E3:CE:30] Client C569752A-45BD-4FE2-B621-3904E69C22D0 attempting to verify, but we are not paired; rejecting client +1ms
EventedHTTPServer [::ffff:192.168.2.43] HTTP Response is finished +2ms
EventedHTTPServer [::ffff:192.168.2.43] Client connection closed +4ms
EventedHTTPServer [::ffff:192.168.2.43] HTTP connection was closed +1ms
EventedHTTPServer [::ffff:192.168.2.43] HTTP server was closed +1ms
EventedHTTPServer [::ffff:192.168.2.249] New connection from client +5s
EventedHTTPServer [::ffff:192.168.2.249] HTTP server listening on port 40033 +1ms
EventedHTTPServer [::ffff:192.168.2.249] Client connection closed +1ms
EventedHTTPServer [::ffff:192.168.2.249] HTTP connection was closed +0ms
EventedHTTPServer [::ffff:192.168.2.249] HTTP server was closed +1ms

I don´t know what this entry means:

HAPServer [CC:22:3D:E3:CE:30] Client C569752A-45BD-4FE2-B621-3904E69C22D0 attempting to verify, but we are not paired; rejecting client +1ms

**Comment by grzegorz914** [comment]
Same problem here, only dynamicPlatform Accessory respond OK, all others no responding.

**Comment by ramirocaballero** [comment]
I have deleted all accessories and it worked again

**Comment by MikeDeltaHH** [comment]
> 
> 
> I have deleted all accessories and it worked again

You mean the cachedAccessories? Try it without success...

**Comment by MikeDeltaHH** [comment]
I found a solution (for me)... made a rollback to 0.4.53 - Homebridge won´t start because some plugins need version 1.0 so i update to 1.0.4 and my Homebridge is visible again in Homekit... very strange...

**Comment by glowfishDE** [comment]
Probably its some cache TTL or something we run into here ?

**Comment by glowfishDE** [comment]
Totally funny - after almost 1,5 weeks with all my devices being "not reachable" its now suddenly working again. I planned to downgrade to 0.4.53, logged into my homebridge, and all i did was installing todays update for homebridge-config-ui-x v4.19.0 and restarted the docker container.

After it came back on 1.0.4 my devices are online in my iphone again. So i didnt downgrade yet.

Can anyone confirm this ? i will continue to watch this.

**Comment by mtressl** [comment]
During the week I had a similar problem: all accessories in my Home app went "No Response".
I found out it was the Blink XT camera plugin, which gave an error but did not crash (Blink switched to 2FA and the plugin did not yet support that). So I disabled the Blink plugin and all was fine again.
Now the Blink plugin got it's update, and also all my accessories are back to normal.

My guess is there is some timeout or uncaught error handling in the latest HB version that leads to these "No Response" situations that did not happen before.

**Comment by MikeDeltaHH** [comment]
I hope that the homebridge devs can confirm and fix this error...

**Comment by mtressl** [comment]
@MikeDeltaHH it might involve HB to fix this, but in my case the main cause was a malfunctioning plugin tbh.

**Comment by twynne** [comment]
I’m seeing similar... after about 24 hours of running, I start seeing all devices as not responding in Apple Home app. Nothing of any interest in the logs, and a restart of homebridge resolves it every time. 

I have the blink plugin but it’s up to date. Other plugins:

Config UI
Wemo platform
Harmony platform
Dyson pure cool
SSH 
Bandwidth quality
Unifi occupancy

Most of these were in use before the homebridge update with no issues. Blink and Dyson have had recent updates, the rest are fairly ‘old’.

**Comment by mtressl** [comment]
Blink seems fine now after the 2FA update, at least on my side.
Cannot talk about the Dyson plugin. So I'd start looking into the fairly ‘old’ ones - one by one - removing the config from config.json and pasting it into a temp text file. Restart HB and see if something changes. You usually do not need to remove the plugin from HB, since no config, no plugin interaction.

**Comment by twynne** [comment]
Challenge is that the old plugins were there long before the homebridge upgrade and no such issues, so unless you’re implying they have issues with the upgraded homebridge I’d be more inclined to look deeper at homebridge itself. The only plugin added since homebridge upgrade is SSH.

**Comment by MikeDeltaHH** [comment]
Before the homebridge update to 1.0.4 my plugins were also running without any problems and during the troubleshooting I uninstalled one plugin after the other without success! Only after a downgrade to 0.4.53 and a new update to 1.0.4 my Homebridge was visible in Homekit again so that I guess the reason for this is Homebridge.

**Comment by manolo4711** [comment]
Same problem here. Clean Install with homebridge-image.. Also an update to 1.1 did't solve the problem. It's really a strange behavior. Sometimes all devices are accessible in the home app and sometimes not. Especially If I close the home app and open the home app again after a while. 

I installed just two plug in: 
shelly 
sonoff

**Comment by gerarddvb** [comment]
Same problem here. Using nest and harmony. It works for 5 minutes. Tried reinstalling evrything, same result

**Comment by gdotp01** [comment]
Ive got the same issue on V1.1.0. It all appears top work but is not visible in the Home App. I've a number of plugins and some response and other don't after a period of time. I only recently updated from V04.53 due to some dependencies and the same setup was stable previously.

**Comment by Cardo1** [comment]
I’ve been struggling with my devices occasionally becoming unresponsive.  The issue started around the time Homebridge went to version 1.*, though I also moved my HB instance to an Raspberry Pi from a Mac Mini around the same time.  My house has solid walls that block WiFi signals, so I have a few APs dotted around connected via a couple of switches.  Sometimes HB would work, other times it wouldn’t.  I could always access the Pi through the Config, even if the devices were showing as unresponsive.  My usual hub is my HomePod in my kitchen, which can connect to one of two APs.  Sometimes I could access the HB devices remotely, other times I couldn’t.

I think I have finally narrowed it down to the way HB now advertises itself, my network’s config and possibly the way the Pi does its networking.  In the end, the trick was to disable “IGMP Snooping” on all routers/switches/APs.  It appears IGMP Snooping can sometimes break mDNS, especially across different network interfaces on a single device (2.4Ghz/5Ghz/Eth).

I’m probably jinxing it by posting this, but so far so good.
